deploy/servctl_utils/other_command_utils.py

```diff
@@ -79,9 +79,12 @@ def delete_all(deployment_target):
 
 
 def get_disk_names(disk, settings):
-    num_disks = settings.get('{}_NUM_DISKS'.format(disk.upper().replace('-', '_'))) or 1
+    disk_setting_prefix = disk.upper().replace('-', '_')
+    num_disks = settings.get('{}_NUM_DISKS'.format(disk_setting_prefix)) or 1
+    name_suffix = settings.get('{}_DISK_NAME_SUFFIX'.format(disk_setting_prefix)) or ''
```

Comment:
Is it more neatly by using `settings.get(ENV_STRING, default_value)`?

Reply:
this handles the empty string better